package.json

```
@@ -7,7 +7,7 @@
   "license": "MIT",
   "scripts": {
     "build": "bunchee",
-    "test": "NODE_ENV=test dotenv -e .env jest"
+    "test": "NODE_ENV=test dotenv -e .env jest --silent=false"
```

Why do we need this?

```suggestion
    "test": "NODE_ENV=test dotenv -e .env jest"
```